apps/hmc/hmc-cft-hearing-service/aat.yaml

```diff
@@ -9,3 +9,4 @@ spec:
         HMI_BASE_URL: https://hmi-apim.staging.platform.hmcts.net/hmi
         LOGGING_LEVEL_UK_GOV_HMCTS_REFORM_HMC_CONFIG: DEBUG
         LOGGING_LEVEL_UK_GOV_HMCTS_REFORM_HMC_REPOSITORY: DEBUG
+        CFT_HEARING_SERVICE_S2S_AUTHORISED_SERVICES: xui_webapp,hmc_hmi_inbound_adapter,sscs,fis_hmc_api
```

Comment:
civil_service should be added to the whitelist

Reply:
Added in all files

Reply:
added in the hmc-cft-hearing-service.yaml and removed in other env yamls